keyboards/rgbkb/zen/rev1/keymaps/samae/keymap.c

```diff
@@ -0,0 +1,89 @@
+    KC_LSFT,  KC_Z,    KC_X,    KC_C,    KC_V,     KC_B,                      KC_N,               KC_M,     KC_COMM, KC_DOT,  KC_SLSH, RGB_MOD, \
+    KC_LCTL,  KC_LGUI, KC_LALT, RGB_TOG, MO(_NAV), KC_SPC, KC_GESC,  KC_ENT,  LT(_NAV, KC_SPACE), KC_RALT, MO(_NAV), _______, KC_MINS, KC_EQL   \
+  ),
+
+  /* NAV
+   * ,-----------------------------------------.                            .-----------------------------------------.
+   * |  F1  |  F2  |  F3  |  F4  |  F5  |  F6  |                            |  F7  |  F8  |  F9  |  F10 |  F11 | F12  |
+   * |------+------+------+------+------+------|                            |------+------+------+------+------+------|
+   * |      |RGBSAI|RGBVAI|RGBSAD|      |  [   |                            |   ]  | Pgup |  Up  | Pgdn |Insert| Home |
+   * |------+------+------+------+------+------|                            |------+------+------+------+------+------|
+   * |      |RGBHUD|RGBVAD|RGBHUI|      |      |                            |      | Left | Down | Right|Delete| End  |
+   * |------+------+------+------+------+------|                            |------+------+------+------+------+------|
+   * |Shift |      |      |      |      |      |                            |      |      |      | Pause| Back | Next |
+   * |------+------+------+------+------+------+------.              .------+------+------+------+------+------+------|
+   * | Ctrl | GUI  | Alt  |RGBMOD|      |      |      |              | RESET|      |      |      | Mute | VOLUP| VOLDN|
+   * `------------------------------------------------'              '------------------------------------------------'
+   */
+  [_NAV] = LAYOUT( \
+    KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,                     KC_F7,   KC_F8,    KC_F9,   KC_F10,  KC_F11,  KC_F12,  \
+    _______, RGB_SAI, RGB_VAI, RGB_SAD, _______, KC_LBRC,                   KC_RBRC, KC_PGUP,  KC_UP,   KC_PGDN, KC_INS,  KC_HOME, \
+    _______, RGB_HUD, RGB_VAD, RGB_HUI, _______, _______,                   _______, KC_LEFT,  KC_DOWN, KC_RGHT, KC_DEL,  KC_END,  \
+    KC_LSFT, _______, _______, _______, _______, _______,                   _______, _______,  _______, KC_MPLY, KC_MPRV, KC_MNXT, \
+    KC_LCTL, KC_LGUI, KC_LALT, RGB_MOD, _______, _______, _______, RESET,   _______, _______,  _______, KC_MUTE, KC_VOLU, KC_VOLD  \
+  ),
+
+};
+
+#ifdef AUDIO_ENABLE
+float tone_qwerty[][2]     = SONG(QWERTY_SOUND);
+#endif
+
+void persistant_default_layer_set(uint16_t default_layer) {
+  eeconfig_update_default_layer(default_layer);
+  default_layer_set(default_layer);
+}
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+  switch (keycode) {
+    case QWERTY:
+      if (record->event.pressed) {
+        #ifdef AUDIO_ENABLE
+          PLAY_NOTE_ARRAY(tone_qwerty, false, 0);
+        #endif
+        persistant_default_layer_set(1UL<<_QWERTY);
+      }
+      return false;
+      break;
+  }
+  return true;
+}
```

Comment:
None of this is needed, or even used, really. The default layer function is is depricated, as well. 
```suggestion
```

Reply:
Thanks! This piece of config has been in limbo for a year and some, I haven't kept up with the changes nearly enough.